lighthouserc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "ci": {
+    "collect": {
+      "startServerCommand": "pnpm preview",
+      "startServerReadyPattern": "SvelteKit",
```

Comment:
question: why this pattern? it confused me a little bit because I didn't understand what this option should do until I looked it up.

suggestion: maybe we could use the `http://localhost:3000` as the pattern to ensure that this is the address that the server is on?